src/libraries/Common/tests/LongName/ClassDefinedInAssemblyWithAVeryLongName.cs

```diff
@@ -0,0 +1,9 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System
+{
+    public class ClassDefinedInAssemblyWithAVeryLongName
```

Comment:
Should this name be longer than 260 chars?

Reply:
I added a comment with an explanation of why it exists (TLDR: if dll defines no types, it might not get copied to the output folder and we can't load it dynamically)